resources/views/layouts/printable_master.blade.php

```diff
@@ -15,7 +15,12 @@
         <meta name="csrf-token" content="{{ csrf_token() }}">
     </head>
     <body>
-
+    <script type="text/php">
+    if ( isset($pdf) ) {
+        $font = $fontMetrics->getFont("helvetica", "bold");
+        $pdf->page_text(72, 18, "Page: {PAGE_NUM} of {PAGE_COUNT}", $font, 6, array(0,0,0));
```

Comment:
Cool!

Reply:
Apparently there's a limited subset of things it can do, mostly drawing to the canvas. Not sure that can create a malicious PDF.